deploy/examples/common.yaml

```diff
@@ -863,16 +863,6 @@ rules:
 ---
 kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
-metadata:
```

Comment:
these changes should not be removed or else the user needs to replay these permissions if he wants to enable csiaddons, in other words, no harm in keeping these by default.

Reply:
Though I only changed the condition of `if` statements at first, the CI complained that I should execute `make rbac` command and then push the generated changes. Code changes in this file were all generated by `make rbac` command.
Because `csiAddons.enabled` is set to `false` by default, I thought these changes are valid.
If my understanding is wrong, I would appreciate it if you could point it out.

Reply:
@BlaineEXE @travisn @leseb  this could be a problem for the user to enable a few functionalities


* We need to update the documents to create the RBAC and enable the required sidecars
* OR create the RBAC rules by default only configmap change is fine to enable the functionality?

Reply:
It makes sense that these permissions should only be required if csi addons is enabled. If it is enabled, then they would need to update the helm chart anyway to update the RBAC. So it sounds like the issue is that the `make rbac` needs to enable this flag when it generates the common.yaml. @BlaineEXE Can we enable this flag for `make rbac`?

Reply:
@Madhu-1 @travisn @BlaineEXE Thank you for discussing this topic.
> @BlaineEXE Can we enable this flag for make rbac?

Do you mean to change the default value of csi.csiAddons.enabled flag to `true` in `values.yaml` file? If so, I agree with this idea.

@BlaineEXE I would be happy if you could comment on it.

Reply:
CSI addons should remain disabled by default since it requires independent CRDs to be installed. So we will need to find a way to keep the rbac optional in the helm chart based on that setting, but just add it to common.yaml always since we don't have a good way to make rbac more optional in the non-helm case.

Reply:
@travisn @BlaineEXE 
> Just add --set csiAddons.enabled, and you should be good to go.

This worked perfectly! `common.yaml` was reverted to the original state as expected.
I pushed the changes.

Two small things:
- The correct parameter was `csi.csiAddons.enabled`. Setting `csiAddons.enabled=true` did not worked well.
- The correct command to generate `common.yaml` was `makge gen-rbac`, not `make rbac`. Sorry for my mistake.